contrib/core-contract-tests/tests/pox-4/pox_Commands.ts

```diff
@@ -63,6 +64,35 @@ export function PoxCommands(
         r.margin,
       )
     ),
+    // DelegateStackStxCommand
+    fc.record({
+      operator: fc.constantFrom(...wallets.values()),
+      // We can pick a constant from operator wallet.wasDelegatedBy: chain and tuple?
+      stacker: fc.constantFrom(...wallets.values()),
+      // Can call the real to find the current pox cycle and find the max?
```

Comment:
We've discussed about this, and at the end this didn't make it in the PR. After all, is it better to randomly pick from the range [0, 1049] for `start-burn-ht`?

Reply:
Considering the stateful enivronment will run across multiple reward cycles, no. Just left it here to discuss about it and pick the best option (and the best fast-check option to do it, too, being aware of the deterministic nature and shrinking)

Reply:
Can't we query/calculate this using the `simnet` instance that's passed in the `Real`?